### DISCUSSION TITLE
Can't enable certificate, thus can't use Hetty

### DISCUSSION THREAD

**Discussion by WedLauren00** [dstotijn/hetty#113] [Q&A]
Hello, while getting started with Hetty I've encountered the following problem: 
Every time I request a page within my browser (Firefox) I see an alert (Warning: Potential Security Risk Ahead)
which introduces an obatacle to view any page. Next, I tried to find a solution to my problem, and then attempted to 
mirror steps from this article (https://hetty.xyz/docs/guides/trust-root-ca/). Entered commands provided chronologically:

```
└─# hetty cert install
INFO To install the certificate in the system trust store, you might be prompted for your password.
FATAL Command failed. {"error": "failed to install certificate: open /root/snap/hetty/9/.hetty/hetty_cert.pem: no such file or directory"}


└─# hetty --cert /root/snap/hetty/9/.hetty/hetty_cert.pem 
INFO [main] Hetty (v0.7.0) is running on :8080 ...
INFO [main] Get started at http://localhost:8080                                              
ERROR [proxy] Securing client connection failed. {"error": "handshake error: remote error: tls: unknown certificate authority", "remoteAddr": "127.0.0.1:40682"}
^C2022/09/11 17:21:11 INFO [main] Shutting down HTTP server. Press Ctrl+C to force quit.

└─# hetty cert install
INFO To install the certificate in the system trust store, you might be prompted for your password.
FATAL Command failed. {"error": "failed to install certificate: failed to execute sudo: fork/exec /usr/bin/sudo: permission denied"}
```

I didn't find any topics with hints for further actions, so thank you in advance for any help or your assistance.

**Comment by who8mypnuts** [comment]
I do see that this post is over a year old, but it will be beneficial to anyone else who runs into issues like I did.

Copy the hetty cert to a non-hidden directory. 
`cp /root/snap/hetty/9/.hetty/hetty_cert.pem /home/userName/Documents/` or something else.

Then, in your browser (I'm using vivaldi), install the certificate as an "Authority." Use this in your URL browser to get you there more quickly `vivaldi://settings/certificates?search=certi`
![2023-10-12_18-45](https://github.com/dstotijn/hetty/assets/23723456/b0127acd-10b2-4e29-b70f-d23f04480db2)

Click "Authorities" and then "Import"
Finally, trust all three by clicking each checkbox.

Enable your proxy, and you're good to go!
